tools/buildtools/ambertools/mmpbsa_mmgbsa.xml

```diff
@@ -124,6 +125,10 @@ with open("$parameteroutfile",'w+') as f:
     <data format="txt" name="resultoutfile" label="${tool.name}: Statistics"/>
     <data format="txt" name="decompoutfile" label="${tool.name}: Decomposition Statistics"/>
     <data format="txt" name="parameteroutfile" label="${tool.name}: parameter output"/>
+    <collection name="output_collection_for_debug" type="list" label="${tool.name} mdout debug " format="txt">
+        <discover_datasets pattern="(?P&lt;designation&gt;.+)\.mdout.*" format="txt" visible="false"/>
```

Comment:
I think this is now failing linting.

Reply:
Thanks, I must check that out! I changed to Draft/WIP as making a few more bigger changes. That I might get completed  today